src/Workspaces/Remote/Core/AbstractAssetProvider.cs

```diff
@@ -98,11 +98,11 @@ public async Task<DocumentInfo> CreateDocumentInfoAsync(Checksum documentChecksu
         {
             var documentSnapshot = await GetAssetAsync<DocumentStateChecksums>(documentChecksum, cancellationToken).ConfigureAwait(false);
             var documentInfo = await GetAssetAsync<DocumentInfo.DocumentAttributes>(documentSnapshot.Info, cancellationToken).ConfigureAwait(false);
-            var serializableSourceText = await GetAssetAsync<SerializableSourceText>(documentSnapshot.Text, cancellationToken).ConfigureAwait(false);
+            var sourceText = await GetAssetAsync<SourceText>(documentSnapshot.Text, cancellationToken).ConfigureAwait(false);
```

Comment:
everyone who used to read out a SerializableSourceText (which all then call GetTextAsync on it), jsut read out a SourceText instead now.